src/Panel/Panel/Panel.jsx

```diff
@@ -351,6 +386,9 @@ export class Panel extends React.Component {
         {titleBar}
         <div
           className="body"
+          tabIndex="0"
+          ref={this.onBodyRef.bind(this)}
```

Comment:
No binding needed as you don't use `this` inside the function.

Reply:
Hoppla :eyes: Removed in 74974c0